lib/itax_code/parser.rb

```diff
@@ -15,7 +15,7 @@ class Parser
 
     # @param [String] tax_code
     # @param [Utils]  utils
-    def initialize(tax_code, utils = Utils.new)
+    def initialize(tax_code, utils = ItaxCode::Utils)
```

Comment:
The prefix is unnecessary here, as you're already inside the ItaxCode module.

```suggestion
    def initialize(tax_code, utils = Utils)
```